vendor/modules.txt

```diff
@@ -576,11 +570,8 @@ google.golang.org/appengine/urlfetch
 # google.golang.org/genproto v0.0.0-20200815001618-f69a88009b70
 ## explicit
 google.golang.org/genproto/googleapis/api/annotations
-google.golang.org/genproto/googleapis/api/httpbody
 google.golang.org/genproto/googleapis/rpc/status
-google.golang.org/genproto/protobuf/field_mask
 # google.golang.org/grpc v1.29.1
-## explicit
 google.golang.org/grpc
```

Comment:
Is something else pulling this in now?

Reply:
Yes I figured out that it is now pulled by the protobuf stack.